Shared/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTracking</key>
+	<false/>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeCoarseLocation</string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAppFunctionality</string>
```

Comment:
Here is the "[Privacy Nutrition Label](https://developer.apple.com/app-store/app-privacy-details/)" for the 100.x samples app. Since App Store Connect reviewers allowed us to publish the old app without adding this entry, I'm a little bit hesitant to add it. But if that is what Apple requires us to do now, then let's do it.

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/4d78ee29-0920-41b6-95a7-d64451a10dca)

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/15a9938f-7b03-452c-9e00-d098b1395167)

From the doc

> “Collect” refers to transmitting data off the device in a way that allows you and/or your third-party partners to access it for a period longer than what is necessary to service the transmitted request in real time.

I can see both sides on whether this app is counted as collecting location data or not. 🎭

Reply:
@dg0yal Do you have any thoughts on this?

Reply:
[This](https://developer.apple.com/app-store/app-privacy-details/) is interesting too:
![Screenshot 2023-11-09 at 9 54 00 AM](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/74069582/dcb554ca-1547-43cf-bfaf-395e70da1144)

Reply:
No we don't "collect" data as per this definition.

Reply:
@yo1995 Apple says this too, "For example, if you collect data solely for the purpose of app functionality, declare the data on your label and indicate that it is only being used for that purpose."

Reply:
Yeah, but it doesn't change the definition of "collect"
For instance, you could be sending device location to a server to track a user's run. That would be data collection for app functionality

Reply:
I see, good point. Ok, sounds like I can remove this item from the privacy manifest then.

Reply:
So if we had a sample that showed how to take your current location and sent it to a feature service, then we would need to report that in the nutrition label, right? Same if we had a sample that geocoded your current location, right? 

So we probably just need to consider this when implementing samples and how samples might affect what we "collect".

Reply:
storing curring location in a feature service, yes i think so

geocoding your current location, probably not (does not satisfy the _allows you and/or your third-party partners to access it for a period longer than what is necessary to service the transmitted request in real time_   )

Reply:
That makes sense thank you. The only sample where we get the user's location is the show device location. In that sample, we are simply showing the user's location and are not transmitting it anywhere. So no need for this location item in our privacy manifest at this moment.